build/RunTestsOnHelix.cmd

```diff
@@ -47,9 +47,4 @@ dotnet nuget remove source dotnet-libraries-transport --configfile %TestExecutio
 dotnet nuget remove source dotnet-tools-transport --configfile %TestExecutionDirectory%\nuget.config
 dotnet nuget remove source dotnet-libraries --configfile %TestExecutionDirectory%\nuget.config
 dotnet nuget remove source dotnet-eng --configfile %TestExecutionDirectory%\nuget.config
-dotnet nuget list source --configfile %TestExecutionDirectory%\nuget.config
-
-robocopy %HELIX_CORRELATION_PAYLOAD%\t\TestExecutionDirectoryFiles\ .\ testAsset.props
-set TestPackagesRoot=%CD%\assets\testpackages\
-dotnet build assets\testpackages\Microsoft.NET.TestPackages.csproj /t:Build -p:VersionPropsIsImported=false
-robocopy .\assets\testpackages\testpackages %TestExecutionDirectory%\TestPackages /s
\ No newline at end of file
+dotnet nuget list source --configfile %TestExecutionDirectory%\nuget.config
```

Comment:
Should this be moved up before we add sources?
Based on this old comment there used to be concurrency issues doing it this way.

REM avoid potetial concurrency issues when nuget is creating nuget.config 

https://github.com/dotnet/sdk/pull/29376/files

Reply:
These lines are the same:
![image](https://github.com/dotnet/sdk/assets/17788297/9274ecf1-9c60-404c-843a-a92d70789ace)

The one for which you are referring is still there. In the original PR, it was just moved down and the comment removed. I didn't revert that since it isn't a functional difference.

Reply:
Thank you, apologies for not seeing that